src/libstd/sys/unix/fs.rs

```diff
@@ -269,6 +269,9 @@ impl File {
             libc::open(path.as_ptr(), flags, opts.mode)
         }));
         let fd = FileDesc::new(fd);
+        // Even though we open with the O_CLOEXEC flag, still set CLOEXEC here,
```

Comment:
In some cases an older version of any OS might not implement atomic option even if the constant is correct.